common/assets/scss/utilities/_typography.scss

```diff
@@ -4,3 +4,6 @@
 .app-font-weight-normal {
   font-weight: normal;
 }
+.app-green {
+  color: govuk-colour("green")
+}
```

Comment:
Could we add a new line at the end of this file. There's a handy [editorconfig plugin](https://editorconfig.org/#download) for most IDEs that will take care of this for you in future based on [the rules we've defined](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/71da62ab17e646eaffaaee1ad2a4c82c4d14f502/.editorconfig#L7) in this repo :)